Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -1377,9 +1378,6 @@ private static T FindImmediateChild<T>(HierarchyNode parent)
                     // Add all our files into our analyzer
                     var files = AllVisibleDescendants.OfType<PythonFileNode>().Select(f => f.Url).ToArray();
                     await analyzer.AnalyzeFileAsync(files);
-                    // Ensure any open files that belong to our project are associated
```

Comment:
What happens now when you have a file open that lives in a project and then open that project?